Source/WebKit/WebProcess/WebPage/DrawingArea.h

```diff
@@ -178,6 +178,10 @@ class DrawingArea : public IPC::MessageReceiver, public WebCore::DisplayRefreshM
     const DrawingAreaType m_type;
     DrawingAreaIdentifier m_identifier;
     WebPage& m_webPage;
+    bool m_shouldScaleViewToFitDocument;
```

Comment:
Initialize this to `{ false }` and move it next to `m_isScalingViewToFitDocument` to optimize padding.

Reply:
Fixed.